msprime/trees.py

```diff
@@ -818,8 +818,41 @@ def load(path):
     return TreeSequence.load(path)
 
 
-def load_tables(*args, **kwargs):
-    return TreeSequence.load_tables(*args, **kwargs)
+def load_tables(nodes, edges, sequence_length=None, migrations=None, sites=None,
+                mutations=None, provenances=None):
+    """
+    Loads a tree sequence from the table objects provided.  If sequence_length
+    is 0 or not provided, it is inferred to be equal to the largest right value
+    in the edges.
+
+    :param NodeTable nodes: The NodeTable.
+    :param EdgeTable edges: The EdgeTable.
+    :param sequence_length: The length of the sequence.
+    :param MigrationTable migrations: The MigrationTable.
+    :param SiteTable sites: The SiteTable.
+    :param MutationTable mutations: The MutationTable.
+    :param ProvenanceTable provenances: The ProvenanceTable.
+    :return: A :class:`msprime.TreeSequence` consistent with the tables.
+    :rtype: TreeSequence
+    """
+    if nodes is None:
```

Comment:
I don't think there's much point in doing these checks here, as it's all done already by the low-level module. I think it's better to just stick with the existing way of packing the (*args, **kwargs) --- otherwise, we're doing the same thing twice in two places.